### PR TITLE
economy changes fixes

### PR DESCRIPTION
**Pull request by Manuel-777**
Just a draft to work on and polish. Needs to handle more cases and fix the new contexts / convert.

### CODE REVIEW

**Comment by lusbenjamin** [comment]
### Current WIP
image

### TODO
- [ ] test gaining orbs
- [ ] test spending orbs
- [ ] test new player experience

File: src/window_main/economy.js [hunk 1]
```diff
@@ -111,12 +103,18 @@ function localDayDateFormat(date) {
...
 
-function getReadableReward(rewardCode) {
-  return rewardCodeMap[rewardCode] || rewardCode;
+function getReadableTrack(trackCode) {
+  return trackCodeMap[trackCode] || getReadableCode(trackCode);
 }
 
-function getReadableTrack(trackCode) {
-  return trackCodeMap[trackCode] || trackCode;
+// quick and dirty generic pretty formatting
```
Comment (Manuel-777) [inline src/window_main/economy.js]: I like dirty and pretty ..

**Comment by Manuel-777** [comment]
Maybe @AnnanFay can throw us a bone to test draft pools/share a log? Besides that it looks great!

Once we are through with the fixes we can start thinking how to squeeze more details of the transactions in our current model, or simply migrate to a new one.

**Comment by Lego6245** [comment]
This should (mostly) resolve #658

**Comment by AnnanFay** [comment]
> Maybe @AnnanFay can throw us a bone to test draft pools/share a log?

Sorry no logs, I've not drafted in the last few days and probably won't until ~Tuesday. I'll post one then unless someone beats me to it.

**Pull request merged**